internal/api/v1/application/part.go

```diff
@@ -56,8 +56,8 @@ func GetPart(c *gin.Context) apierror.APIErrors {
 	partName := c.Param("part")
 	logger := requestctx.Logger(ctx)
 
-	if partName != "values" && partName != "chart" && partName != "image" {
-		return apierror.NewBadRequestError("unknown part, expected chart, image, or values")
+	if partName != "manifest" && partName != "values" && partName != "chart" && partName != "image" {
+		return apierror.NewBadRequestError("unknown part, expected chart, manifest, image, or values")
```

Comment:
This `if` statement is becoming a bit long, and it's better to log/return the wrong part.

What about this?

```golang
switch partName {
// valid parts
case "manifest", "values", "chart", "image":
default:
	return apierror.NewBadRequestErrorf("unknown '%s' part, expected chart, manifest, image, or values", partName)
}
```

Reply:
Agreed.

Reply:
Done, commit 281c55964128a18f8a6bcf3b4d75b06e80395032